app/core/config.py

```
@@ -82,4 +127,80 @@ def validate_aws_config(cls) -> bool:
+    logger = logging.getLogger(__name__)
+    system_prompts.clear()  # Clear existing prompts
+     
+    for agent_name, agent_config in config.AGENT_CONFIGS.items():
+         prompt_path = agent_config.get("system_prompt_path")
+         if prompt_path:
+            try:
+                with open(prompt_path, "r", encoding="utf-8") as f:
+                    logger.info(f"Loaded system prompt for agent '{agent_name}' from {prompt_path}")
+                    system_prompts[agent_name] = f.read()
+            except FileNotFoundError:
+                raise RuntimeError(f"System prompt file not found for agent '{agent_name}': {prompt_path}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)